### ISSUE TITLE
Compilation failure with 5.25.10

### ISSUE THREAD

**Issue by eserte** [tokuhirom/Perl-Lexer#8]
With perl 5.25.10 I see the following compilation error (linux, freebsd, darwin):
```
Building Perl-Lexer
cc -I/opt/perl-5.25.10/lib/5.25.10/x86_64-linux/CORE -DVERSION="0.22" -DXS_VERSION="0.22" -fPIC -c -fwrapv -fno-strict-aliasing -pipe -fstack-protector-strong -I/usr/local/include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -D_FORTIFY_SOURCE=2 -O2 -o lib/Perl/Lexer.o lib/Perl/Lexer.c
In file included from lib/Perl/gen/token_info-latest.h:1:0,
                 from lib/Perl/gen/token_info_map.h:250,
                 from lib/Perl/Lexer.xs:16:
lib/Perl/gen/perly-latest.h:179:1: error: unterminated comment
 /*
 ^
error building lib/Perl/Lexer.o from 'lib/Perl/Lexer.c' at /opt/perl-5.25.10/lib/5.25.10/ExtUtils/CBuilder/Base.pm line 174.
```

**Comment by charsbar** [comment]
Thanks for spotting. Shipped 0.23

**Comment by eserte** [comment]
Confirmed, looks good with 0.23.
